docs/relnotes.md

```diff
@@ -7,18 +7,62 @@ hide:
 
 This is the list of changes to Taipy releases as they were published.
 
-## Community edition: 1.2 (In progress)
+## Community edition: 2.0 (In progress)
+
+Published on TO_BE_DEFINED.
+
+[`taipy` 2.0](https://pypi.org/project/taipy/2.0.0/) contains the latest
+[`taipy-config` 1.0](https://pypi.org/project/taipy-config/1.0.0/),
+[`taipy-gui` 2.0](https://pypi.org/project/taipy-gui/2.0.0/),
+[`taipy-core` 2.0](https://pypi.org/project/taipy-core/2.0.0/) and
+[`taipy-rest` 2.0](https://pypi.org/project/taipy-rest/2.0.0/) packages.
+
+#### Features
+
+**`taipy-core`** 2.0.0
+
+   - TODO SQLDataNodes
+   - TODO SQLTableDatanode
+   - TODO JSONDataNodes
+   - In standalone mode, a job whose status is `SUBMITTED`, `PENDING`, or `BLOCKED`, can be canceled. When
+     canceling the job, its subsequent jobs will be abandoned and their statuses will be set to `ABANDONED`. When
+     the cancel method is called on a job with its status being either `RUNNING`, `COMPLETED`, or `SKIPPED`, its
+     subsequent jobs will be abandoned while its status remains unchanged. A job whose status is `FAILED`,
+     `CANCELED`, or `ABANDONED`, cannot be canceled.
```

Comment:
I would have removed the comma before 'cannot'.
And rewrite 'cannot' to 'can not'.

Reply:
Good question about cannot.
I just found [this](https://www.grammarly.com/blog/cannot-or-can-not/#:~:text=Here%20is%20a%20quick%20summary,.%20.%20.%20but%20also.%E2%80%9D) on grammarly.
I would keep the cannot.

Reply:
See https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference#:~:text=Both%20cannot%20and%20can%20not,general%2C%20it%20is%20somewhat%20informal.

"Cannot has been in use since the 15th century"... 
Cannot is the way to go. Please forget my comment.